services/avs/mgmt/2020-03-20/src/lib.rs

```diff
@@ -0,0 +1,27 @@
+pub mod models;
+pub mod operations;
+
+pub const API_VERSION: &str = "2020-03-20";
+
+pub type Error = Box<dyn std::error::Error + Send + Sync>;
+pub type Result<T> = std::result::Result<T, Error>;
+
+pub struct Configuration {
+    pub api_version: String,
```

Comment:
Is there a reason to make this a field? This value should always be set to `API_VERSION` no? Maybe this should be an associated constant?

Reply:
Still not sure if exposing this as raw fields as opposed to getter/setter methods is the right way to go though.

Reply:
I don't have a strong opinion here.

Reply:
I'm open to using getter/setter methods. I'll keep it in mind when we make changes here.